cmd/demos/event-replay/main.go

```diff
@@ -0,0 +1,88 @@
+package main
```

Comment:
Cool!! Can you also add a readme file that says how to set this up?

Reply:
Done